.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,60 @@
+name: Code quality checks
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+    branches:
+      - master
+      - stable/*
+    tags:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
+  pull_request:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
+  workflow_dispatch:
+
+jobs:
+  isort:
+    name: Check import sorting
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install OS dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
```

Comment:
Don't think we need these?

Reply:
why was this resolved?

Reply:
hmm, something must have gone wrong with pushing, I'll fix it